src/core_plugins/metrics/public/components/icon_select.js

```diff
@@ -27,15 +27,21 @@ class IconOption extends Component {
 
   render() {
     const icon = this.props.option.value;
-    const title = this.props.option.label;
+    const label = this.props.option.label;
     return (
       <div className={this.props.className}
         onMouseEnter={this.handleMouseEnter}
         onMouseDown={this.handleMouseDown}
         onMouseMove={this.handleMouseMove}
-        title={title}>
-        <span className="Select-value-label">
-          <i className={`vis_editor__icon_select-option fa ${icon}`}></i>
+      >
+        <span
+          className="Select-value-label"
+          aria-label={`${label} icon`}
```

Comment:
We should use aria-label instead of title. And to provide context, we'll also append the word "icon".